### DISCUSSION TITLE
Make invoice template customizable

### DISCUSSION THREAD

**Discussion by Moini** [baimard/gestion#185] [Ideas]
The default invoice template does not allow for logos, multi-line addresses, custom font sizes/colors, envelope-compatible documents,... 

Please provide a template that can be styled according to the user's needs and legal / postal requirements.

This is a long-term project, as it requires to choose a template language and design. 

Maybe HTML would be suitable? At least that converts well to pdf and can be styled / layouted.

**Comment by baimard** [comment]
This requested feature have been included in the roadmap.

Please see discussions : https://github.com/baimard/gestion/discussions

**Comment by Moini** [comment]
Merci. Is there a way to just move this, instead of asking users to copy-paste?

**Comment by baimard** [comment]
done
